modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetRequest.java

```diff
@@ -63,6 +65,8 @@
  * This is a class for representing a request to be sent to a target.
  */
 public class TargetRequest {
+
+	private static final Log log = LogFactory.getLog(TargetRequest.class);
```

Comment:
minor formatting issue. please align.

Reply:
Yes, It is already aligned at next commit.

Reply:
Thanks.